### DISCUSSION TITLE
dodać coś aby bot zapamietywał słówka sam na następne sesje instalingu

### DISCUSSION THREAD

**Discussion by marguz11** [Endiasz/Instaling-Bot#8] [Ideas]
chodzi o to że jak sie wyłączy bota albo zacznie następny instaling to bot automatycznie zapomina słowa których sam sie nauczył 
więc przydało by sie takie coś dodane

**Comment by Endiasz** [comment]
Siemka. Coś na ten styl miało być wprowadzone domyślnie i automatycznie, jednak chęci i umiejętności mi zabrakło w tamtym czasie, ale na końcu każdego dnia czy jak to się tam nazywało pojawiają ci się wszystkie słówka, wystarczy je przekopiować do bota ( pojawia się na końcu wszytkie słowa i dodać ję do bota poprzej dodaj słowa).
